.circleci/config.yml

```
@@ -6,10 +6,10 @@ workflows:
   package-and-push-chart-on-tag:
     jobs:
       - architect/push-to-app-catalog:
-          name: "package and push {CHART-NAME} chart"
+          name: "package and push helm-2to3-migration chart"
```

```suggestion
          name: "push-helm-2to3-migration-to-default-catalog"
```

I think this is the current naming we're using. Should we use the default catalog because its used by chart-operator?

Also please bump the orb version to the latest (0.8.3).